### PR TITLE
Release 7.0.1

### PR DESCRIPTION
**Pull request by acampos1916**
**Fixes**
Remove unknown function call allowRecurring (#1004)
Add Paybright to open invoice list (#1009)
Adding Cartes Bancaires to supported payment methods (#1015)
[PW-4538] Using type-brand structure for giftcards (#1016)
Update the time calculation between offer closed and created (#1018)
Removing unused AdyenRequestMerchantSessionInterface (#1021)
Removing trailing commas from JS files (#1019)
Setting Pay by Mail configuration to use store ID (#1013)
Attempt to create and mount the component for all HPP methods (Support for Blik) (#1022)
Allowing zero sub totals in Helper/PaymentMethods.php (#1028)
Remove storing additionalData for banktransfer_iban and boleto (#1032)

### CODE REVIEW

**Review by msilvagarcia — DISMISSED** [review]

**Review by msilvagarcia — APPROVED** [review]

**Review by AlexandrosMor — DISMISSED** [review]

**Review by peterojo — APPROVED** [review]

**Comment by cyattilakiss** [comment]
@AlexandrosMor version bump is missing

File: Helper/PaymentMethods.php [hunk 1]
```diff
@@ -209,13 +217,13 @@ protected function getCurrentPaymentAmount()
 
         $total = (float)$total;
 
-        if ($total > 0) {
+        if ($total >= 0) {
```
Comment (cyattilakiss) [inline Helper/PaymentMethods.php:220]: what is the reason behind thins change? I think we don't support 0 cart amount.

File: Model/Cron.php [hunk 2]
```diff
@@ -415,8 +415,7 @@ private function shouldSkipProcessingNotification($notification)
                 // OFFER_CLOSED notifications needs to be at least 10 minutes old to be processed
                 $offerClosedMinDate = new \DateTime('-10 minutes');
                 $createdAt = \DateTime::createFromFormat('Y-m-d H:i:s', $notification['created_at']);
-
-                $minutesUntilProcessing = $createdAt->diff($offerClosedMinDate)->i;
+                $minutesUntilProcessing = ($createdAt->format('U') - $offerClosedMinDate->format('U')) / 60;
```
Comment (cyattilakiss) [inline Model/Cron.php:418]: what was the issue with the previous?

Comment (AlexandrosMor) [reply Model/Cron.php:418]: The usage of ->i is not accurate to calculate the minutes difference between the two DateTime objects created and offer closed . The "i" attribute doesn't take into account the hours and the days in DateInterval.
Using $minutesUntilProcessing = ($createdAt->format('U') - $offerClosedMinDate->format('U')) / 60; Is taking the account the days and the hours

Comment (cyattilakiss) [reply Model/Cron.php:418]: ah got it, thanks for the explanation

**Review by AlexandrosMor — APPROVED** [review]

**Review by peterojo — APPROVED** [review]

**Pull request merged**